### PR TITLE
chore(#8190): remove configuration wizard

### PR DESCRIPTION
**Pull request by latin-panda**
# Description

- Removes the configuration wizard
- Removed related code of configuration wizard in e2e tests

medic/cht-core#8190

Note: **Do not merge - this is for 4.3.0**

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs

If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8190-remove-configuration-wizard/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8190-remove-configuration-wizard/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8190-remove-configuration-wizard/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by latin-panda** [comment]
Hi @dianabarsan! Can you please have a look when you have time? This is for v4.3.0 so there is no rush :)

File: webapp/src/ts/app.component.ts [hunk 1]
```diff
@@ -568,16 +566,7 @@ export class AppComponent implements OnInit, AfterViewInit {
         this.globalActions.setShowPrivacyPolicy(privacyPolicy);
         return { privacyPolicy, accepted };
```
Comment (dianabarsan) [inline webapp/src/ts/app.component.ts:567]: Is this result used anywhere anymore?

**Review by dianabarsan — APPROVED** [review]
Kill it with fire! 
One question inline.

File: webapp/src/ts/app.component.ts (hunk 1 above)
Comment (latin-panda) [reply webapp/src/ts/app.component.ts]: Good catch! Removed

**Pull request merged**